htmresearch/algorithms/faulty_temporal_memory.py

```diff
@@ -58,6 +49,7 @@ def killCells(self, percent = 0.05):
     time you call this method a permutation list is set up. Calls change the
     number of cells considered dead.
     """
+
```

Comment:
I think `killCells` needs to edit `self.activeSegments` and `self.matchingSegments`.

Something like

~~~python
self.activeSegments = [s for s in self.activeSegments if s.cell not in self.deadCells]
self.matchingSegments = [s for s in self.matchingSegments if s.cell not in self.deadCells]
~~~